raster/r.random/local_proto.h

```diff
@@ -48,7 +48,7 @@ void get_stats(struct rr_state *);
 int execute_random(struct rr_state *);
 
 /* support.c */
-int make_support(struct rr_state *, int, double);
+int make_support(struct rr_state *, int, double, long);
```

Comment:
long is not portable: long is 32 bit on 32 bit systems and 64 bit on 64 bit systems, on MS Windows it is always 32 bit. Use grass_int64 if you need a portable 64 bit signed integer type.

Reply:
The original code had long, most likely because `G_srand48(long seedval)` uses long and G_srand48_auto() returns long. So, do we want to change it in the library?

I would say yes, but open a new issue for that. I could merge this as is and the fix can be done with the library fix. Wha do you think?